integration/utils/contLoadGenerator.go

```diff
@@ -411,3 +403,186 @@ func (lg *ContLoadGenerator) sendTx(workerID int) {
 		wg.Wait()
 	}
 }
+
+func (lg *ContLoadGenerator) sendAddKeyTx(workerID int) {
+	// TODO move this as a configurable parameter
+	numberOfKeysToAdd := 40
+	blockRef, err := lg.blockRef.Get()
+	if err != nil {
+		lg.log.Error().Err(err).Msgf("error getting reference block")
+		return
+	}
+
+	lg.log.Trace().Msgf("getting next available account")
+
+	acc := <-lg.availableAccounts
+	defer func() { lg.availableAccounts <- acc }()
+
+	lg.log.Trace().Msgf("creating add proposer key script")
+	keys := make([]*flowsdk.AccountKey, 0)
+	for i := 0; i < numberOfKeysToAdd; i++ {
+		keys = append(keys, lg.serviceAccount.accountKey)
+	}
+
+	addKeysTx, err := lg.scriptCreator.AddKeysToAccountTransaction(*(acc.address), keys)
```

Comment:
It seems like we are constantly dereferencing this address pointer -- can we just make it a non-reference `flow.Address`?

Reply:
I'll address this in the next PR (loader refactor)